### PR TITLE
update publish script

### PR DESCRIPTION
**Pull request by charlierudolph**

### CODE REVIEW

**Review by hugoesthere — APPROVED** [review]
I'm assuming this is to be shipped before #255 right?

File: bin/publish [hunk 1]
```diff
@@ -96,13 +96,13 @@ function update-exocom-package-versions
...
-      files: ['features/**/*.feature', 'exosphere-shared/**/*.yml']
-      replace: /bus:(\s*)type: exocom(\s*)version: \d+.\d+.\d+/g
+      files: ['**/*.feature', 'exosphere-shared/**/*.yml']
+      replace: /bus:\n(\s*)type: exocom\n(\s*)version: \d+.\d+.\d+/g
       with:
         """
         bus:
-          type: exocom
-          version: #{@target-version}
+        $1type: exocom
```
Comment (kevgo) [inline bin/publish:104]: Are you sure this change is correct?

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,718 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (kevgo) [inline yarn.lock]: Why do you add this file here?

File: bin/publish (hunk 1 above)
Comment (charlierudolph) [reply bin/publish:104]: `$1` is the captured whitespace in front of type. I ran this function locally and it appeared correct

**Comment by charlierudolph** [comment]
The top level was missing a `yarn.lock`. I'll break this into its own PR.

File: yarn.lock (hunk 2 above)
Comment (charlierudolph) [reply yarn.lock]: The top level was missing a yarn.lock. I'll break this into its own PR.

**Pull request merged**